src/main/kotlin/dev/architectury/idea/inspection/ImplementExpectPlatformFixDialog.kt

```diff
@@ -40,15 +41,17 @@ class ImplementExpectPlatformFixDialog(
     init {
         title = ArchitecturyBundle["inspection.implementExpectPlatform.single", platform]
         init()
-        destinationBox.setData(project, defaultDirectory, EditorComboBox(""))
-        destinationBox.comboBox.addItemListener {
-            isOKActionEnabled = (it.item as? DirectoryChooser.ItemWrapper)?.directory != null
+        val comboBox = EditorComboBox("").also { ecb ->
+            ecb.addItemListener {
+                isOKActionEnabled = (it.item as? DirectoryChooser.ItemWrapper)?.directory != null
+            }
         }
```

Comment:
```suggestion
        val comboBox = EditorComboBox("")
        comboBox.addItemListener {
            isOKActionEnabled = (it.item as? DirectoryChooser.ItemWrapper)?.directory != null
        }
```
No need for also (I can also commit this change myself after merging)